imports/api/PointCalculationUtils.js

```diff
@@ -0,0 +1,106 @@
+import { Constants } from './Constants';
+import { NewGameUtils } from './NewGameUtils';
+
+export var PointCalculationUtils = {
+	jpn: {
+		dealin_delta,
+		selfdraw_delta,
+		mistake_delta
+	}
+};
+
+function dealin_delta(points, fu, winnerWind, loserWind, riichiSticks) {
```

Comment:
Would it hurt to put comments in your code additions?  It makes it faster for others to review, and easier for others to maintain.  I think we should probably start including function headers as well just to make things easier to understand.